### ISSUE TITLE
Working example

### ISSUE THREAD

**Issue by jarheadfa** [Kyubyong/tacotron#54]
Hi, what is a working example on the bible dataset? Does the last commit works on this dataest?

**Comment by keicoon** [comment]
Yes. I tested this dataset in `b95b1682250d8b6d2932ce0c37f284e503552119`.
